packages/incompatible-packages/lib/status-icon-component.js

```diff
@@ -14,7 +14,7 @@ export default class StatusIconComponent {
   render () {
     return (
       <div className='incompatible-packages-status inline-block text text-error'>
-        <span className='icon icon-bug'></span>
+        <span className='icon icon-bug' />
```

Comment:
The ending `</span>` isn't allowed to be omitted in valid HTML, is etch fixing this or are we just relying on Chrome to fix it for us (which should be fixed)?

Reply:
Good point! in this case I'm pretty sure that this code get transpiled (probably by babel) into `etch` calls to create DOM elements, since this is standard JSX.

Probably @nathansobo or @as-cii know better :)